apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/extract/RifFilesProcessorTest.java

```diff
@@ -38,6 +38,7 @@
 
 /** Unit tests for {@link RifFilesProcessor}. */
 public final class RifFilesProcessorTest {
+  // TODO: Replace with junit5 extension and upgrade class to junit5
```

Comment:
do we still need this TODO?

Reply:
The couple places we use Rules I left running on Junit4 until I researched more about extensions since Rules are totally replaced; however I think the rules that expect no exceptions like this one could maybe be removed since junit automatically fails if an exception is thrown, and it seems redundant to me.

Reply:
Ended up removing it after a team discussion in engineering sync.